app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventPresenter.java

```diff
@@ -53,11 +58,100 @@ private boolean verify() {
         return true;
     }
 
+    private boolean findEmptyLink(String uri) {
+        if (uri == null)
+            return false;
+        else return !("".equals(uri));
+    }
+
+    private void resolveEmptyLinks() {
```

Comment:
Rename to `nullifyEmptyFields`, and make it protected so that we can test this. Also pass event through parameter